src/strand-table/strand-table.html

```diff
@@ -0,0 +1,48 @@
+<!--
+ * @license
+ * Copyright (c) 2015 MediaMath Inc. All rights reserved.
+ * This code may only be used under the BSD style license found at http://mediamath.github.io/strand/LICENSE.txt
+
+-->
+<link rel="import" href="../../bower_components/polymer/polymer.html">
+<link rel="import" href="../shared/behaviors/resolvable.html">
+<link rel="import" href="../shared/behaviors/mixinfindable.html"/>
+<link rel="import" href="../shared/behaviors/templatefindable.html"/>
+<link rel="import" href="../strand-grid-column/strand-grid-column.html">
+<link rel="import" href="../strand-grid-item/strand-grid-item.html">
+<link rel="import" href="../strand-checkbox/strand-checkbox.html">
+<link rel="import" href="../strand-icon/strand-icon.html">
+<link rel="import" href="../strand-loader/strand-loader.html">
+<link rel="import" href="../shared/behaviors/refable.html"/>
+
+<dom-module id="strand-table">
+	<link rel="import" type="css" href="strand-table.css"/>
+	<template>
+		<div id="_overlay">
+			<div id="fixes-flicker">
+				<strand-loader id="loader" style$="{{_loaderStyle}}">
+					<label>Loading...</label>
+				</strand-loader>
+			</div>
+		</div>
+
+		<div id="_constrainer">
+			<div id="header">
+				<div class="_mm_column checkbox" hidden$="{{!selectable}}">
+					<strand-checkbox id="selectAll" on-tap="_toggleAllSelections" state="{{_selectAllState}}"></strand-checkbox>
+				</div>
```

Comment:
Looks good in general, just wanna confirm that the carat/multi-expand functionality was intentionally excluded(?)